config/roguelike_dungeons/settings/loot_armor.json

```diff
@@ -0,0 +1,126 @@
+		{"data": {"name": "minecraft:chainmail_boots", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:chainmail_chestplate", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:chainmail_helmet", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:chainmail_leggings", "meta": 0}, "weight": 10},
+		{"data": {"name": "minecraft:iron_boots", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_chestplate", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_helmet", "meta": 0}, "weight": 8},
+		{"data": {"name": "minecraft:iron_leggings", "meta": 0}, "weight": 8},
+		{"data": {"name": "battlegear2:shield.gold", "meta": 0}, "weight": 15}
+      ]
+    },
+    {"level": 3, "type": "ARMOUR", "each": true, "quantity": 4, "loot": [
+		{"data": {"name": "Railcraft:armor.steel.boots", "meta": 0}, "weight": 15},
+		{"data": {"name": "Railcraft:armor.steel.helmet", "meta": 0}, "weight": 15},
+		{"data": {"name": "Railcraft:armor.steel.legs", "meta": 0}, "weight": 15},
+		{"data": {"name": "Railcraft:armor.steel.plate", "meta": 0}, "weight": 15},
+		{"data": {"name": "minecraft:diamond_boots", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:diamond_chestplate", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:diamond_helmet", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:diamond_leggings", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:iron_boots", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:iron_chestplate", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:iron_helmet", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:iron_leggings", "meta": 0}, "weight": 20},
+		{"data": {"name": "battlegear2:shield.iron", "meta": 0}, "weight": 15}
+    ]
+    },
+    {
+      "level": 4, "type": "ARMOUR", "each": true, "quantity": 5, "loot": [
+		{"data": {"name": "battlegear2:knights_armour.boots", "meta": 0}, "weight": 5},
+		{"data": {"name": "battlegear2:knights_armour.helmet", "meta": 0}, "weight": 5},
+		{"data": {"name": "battlegear2:knights_armour.legs", "meta": 0}, "weight": 5},
+		{"data": {"name": "battlegear2:knights_armour.plate", "meta": 0}, "weight": 5},
+		{"data": {"name": "Railcraft:armor.steel.boots", "meta": 0}, "weight": 15},
+		{"data": {"name": "Railcraft:armor.steel.helmet", "meta": 0}, "weight": 15},
+		{"data": {"name": "Railcraft:armor.steel.legs", "meta": 0}, "weight": 15},
+		{"data": {"name": "Railcraft:armor.steel.plate", "meta": 0}, "weight": 15},
+		{"data": {"name": "minecraft:diamond_boots", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:diamond_chestplate", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:diamond_helmet", "meta": 0}, "weight": 20},
+		{"data": {"name": "minecraft:diamond_leggings", "meta": 0}, "weight": 20},
+		{"data": {"name": "battlegear2:shield.diamond", "meta": 0}, "weight": 15}
```

Comment:
battlegear2 is going to be removed, give out a fully built Tinker's Defense shield here instead.